src/commands.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Copyright (c) 2019 Red Hat, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Red Hat Inc. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * Commonly used commands
+ */
+export namespace Commands {
+ 
+    /**
+     * Auto close tags
+     */
+    export const AUTO_CLOSE_TAGS= 'xml.completion.autoCloseTags';
+
+    /**
+     * Show LSP Standard references
+     */
+    export const SHOW_REFERENCES = 'lsp.show.references';

```

Comment:
there's no such thing as LSP standard references. We should keep xml.show.references until Reference Code lenses are defined at the protocol level.

Reply:
@fbricon updated